datumaro/plugins/vgg_face2_format.py

```diff
@@ -43,16 +57,29 @@ def _load_items(self, path):
 
         for row in landmarks_table:
             item_id = row['NAME_ID']
-            image_path = osp.join(self._dataset_dir, self._subset,
-                item_id + VggFace2Path.IMAGE_EXT)
-            annotations = []
+            label = osp.dirname(item_id)
```

Comment:
Looking at relative item paths, I think, we could use only the first directory name as a label, and only if there is a corresponding label name mapping entry. `item_id` is just a full path. This is a topic for future discussion.

Reply:
If `item.id` is full path, there is a problem in one test.
Let's say `item.id` is not a full path. Convert with label and get `row['NAME_ID'] = label/item.id`. When exporting, we save the full path and get a mismatch.

Reply:
How about using only the first directory as class, and all the rest path as the `id`? It looks like a working solution, if there is a guarantee, that the first directory actually is the label (i.e. we prohibit using an arbitrary directory with images).